web/themes/custom/cj_material/css/styles.css

```diff
@@ -3475,18 +3475,11 @@ a.card, .content-card a.card-holder {
 .chip {
   display: inline-block;
   height: 32px;
-  padding: 0 12px;
-  border-radius: 32px;
-  margin-right: 8px;
-  background: rgba(35, 47, 52, 0.12);
```

Comment:
the "chip" class is also used in the keyword selector on the Job Edit form. Can we check that that still works as expected? Chips have a specific meaning in material design - https://material.io/components/chips/#usage - if this isn't what we're going for, lets change the class rather than changing what a chip looks like.